packages/sit-onyx/src/components/OnyxFileUpload/OnyxFileUpload.vue

```diff
@@ -445,8 +445,8 @@ const shouldShowFileList = computed(() => {
         );
 
         max-height: calc(
-          (var(--onyx-file-upload-max-files) + 0.5) * $file-card-height +
-            (var(--onyx-file-upload-max-files)) * var(--onyx-file-upload-list-gap)
+          (var(--onyx-file-upload-max-files, 3) + 0.5) * $file-card-height +
+            (var(--onyx-file-upload-max-files, 3)) * var(--onyx-file-upload-list-gap)
```

Comment:
```suggestion
          (var(--onyx-file-upload-max-files) + 0.5) * $file-card-height +
            (var(--onyx-file-upload-max-files)) * var(--onyx-file-upload-list-gap)
```

The default value is defined when defining the CSS variable at the top of the CSS in this file